modules/github.test.js

```diff
@@ -262,6 +291,133 @@ describe("github", () => {
+            async (repoName) => {
+                // Arrange
+                const owner = testUtils.randomWord();
+                const echoErrorSpy = jest.spyOn(echo, "error");
+
+                // Act
+                await github.getPullRequests(owner, repoName);
+
+                // Assert
+                expect(echoErrorSpy).toHaveBeenCalled();
+                expect(shellExitSpy).toHaveBeenCalled();
+            }
+        );
+
+        test("given username and repo exists, returns pull requests", async () => {
+            // Arrange
+            const owner = "AndcultureCode";
+            const repoName = "AndcultureCode.Cli";
+
+            const mockedPullRequests = [{ id: random.number() }];
+
+            nock(github.apiRootUrl)
+                .get(getRepoPullRequestsRoute(owner, repoName))
+                .reply(200, mockedPullRequests);
+
+            // Act
+            const results = await github.getPullRequests(owner, repoName);
+
+            // Assert
+            expect(results).not.toBeNull();
+            expect(results.length).toBeGreaterThan(0);
+        });
+    });
+
+    // #endregion getPullRequests
+
+    // -----------------------------------------------------------------------------------------
+    // #region getPullRequestReviews
+    // -----------------------------------------------------------------------------------------
+
+    describe("getPullRequestReviews", () => {
+        test.each([undefined, null, "", " "])(
+            "given owner is %p, it outputs an error and calls shell.exit",
+            async (owner) => {
+                // Arrange
+                const repoName = testUtils.randomWord();
+                const echoErrorSpy = jest.spyOn(echo, "error");
+                const pullNumber = random.number();
+
+                // Act
+                await github.getPullRequestReviews(owner, repoName, pullNumber);
+
+                // Assert
+                expect(echoErrorSpy).toHaveBeenCalled();
+                expect(shellExitSpy).toHaveBeenCalled();
+            }
+        );
+
+        test.each([undefined, null, "", " "])(
+            "given repoName is %p, it outputs an error and calls shell.exit",
+            async (repoName) => {
+                // Arrange
+                const owner = testUtils.randomWord();
+                const echoErrorSpy = jest.spyOn(echo, "error");
+                const pullNumber = random.number();
+
+                // Act
+                await github.getPullRequestReviews(owner, repoName, pullNumber);
+
+                // Assert
+                expect(echoErrorSpy).toHaveBeenCalled();
+                expect(shellExitSpy).toHaveBeenCalled();
+            }
+        );
+
+        test("given username and repo exists, returns pull request reviews", async () => {
+            // Arrange
+            const owner = "AndcultureCode";
+            const repoName = "AndcultureCode.Cli";
```

Comment:
Thought - I had to double check the implementation of the function to make sure it wasn't doing more work than I thought it was. This test setup was a bit misleading to me because I thought it was actually calling out to retrieve pull requests for the given account/repo _first_, and then requesting the reviews from there. 

Since the single API call is being mocked, would it be better to randomize the owner/repo name variables and maybe call out that the important piece of the setup is that they are not empty?